conf/messages

```diff
@@ -1096,7 +1096,7 @@ auto.requestDetailedValuation.rateableValue=<b>Rateable Value:</b> £{0}
 
 auto.requestDetailedValuation.future.tabName=Valuation
 auto.requestDetailedValuation.future.rv.caption=Future rateable value (from {0})
-auto.requestDetailedValuation.future.rv.inset=This is not the amount you will pay. It is used to calculate your business rates bill. {0}.
+auto.requestDetailedValuation.future.rv.inset=This will be the rateable value for the property. It is not what you will pay in business rates or rent. Your local council uses the rateable value to calculate the business rates bill. <br> <br> {0}.
```

Comment:
I think it would be neater to extract the estimator link to a new element that's not part of this message, now that they're not in the same paragraph. Then we can get rid of the `<br>`s